src/components/Footer.jsx

```diff
@@ -80,6 +80,7 @@ class SiteFooter extends React.Component {
                 {this.renderLinkIfExists(process.env.PRIVACY_POLICY_URL, 'Privacy Policy')}
                 {this.renderLinkIfExists(process.env.HONOR_CODE_URL, 'Honor Code')}
                 {this.renderLinkIfExists(process.env.Contact, 'Contact')}
+                {this.renderLinkIfExists(process.env.ACCESSIBILITY_URL, 'Accessibility')}
```

Comment:
Can you add this to the tests as well, something similar to https://github.com/mitodl/frontend-component-footer-mitol/blob/master/src/components/__snapshots__/Footer.test.jsx.snap#L38-L45